lib/app/key_customization/key_customization.dart

```diff
@@ -41,24 +40,21 @@ class KeyCustomizationManager {
     }
 
     try {
-      final jsonString = String.fromCharCodes(base64Decode(pref));
-      return json.decode(utf8.decode(jsonString.codeUnits));
+      return json.decode(utf8.decode(pref.codeUnits));
```

Comment:
I think this can just be `json.decode(pref)`

Reply:
fixed!